template.go

```diff
@@ -57,6 +59,13 @@ func readYAMLRequest(opts *Options) error {
 	}
 
 	base := filepath.Dir(opts.ROpts.YamlTemplate)
+	base, err = filepath.Abs(base)
+	if err != nil {
+		return err
+	}
```

Comment:
// Adding a final slash so that the base URL refers to a directory, unless the base is exactly "/".